### ISSUE TITLE
Cannot publish package - authentication problem

### ISSUE THREAD

**Issue by asiloisad** [atom/atom.io#232]
hi,

I am the author of the package [pdf-viewer](https://atom.io/packages/pdf-viewer). About two weeks after the `apm publish patch` command I got the information `Creating new version failed: Requires authentication. Please update your token if you haven't done so recently`. To fix the problem I went to https://github.com/settings/applications and revoked all permissions for Atom, then logged back into https://atom.io/account again granting permissions, then with the `apm login` command I entered a new token into apm console. I repeated this action many times, unfortunately it did not solve my problem. Do you have any idea how I can solve this?

console output for `apm publish patch`::

    Preparing and tagging a new version done
    Pushing v0.1.17 tag done
    Publishing pdf-viewer@v0.1.17 failed
    Creating new version failed: Requires authentication. Please update your token if you haven't done so recently.

the result of `apm publish patch` is the new package version is updated in github, it can be install by `apm install bacadra/atom-pdf-viewer`, but is isn't available in atom package store.

**Comment by asiloisad** [comment]
has any one similar problem now?

**Comment by shellscape** [comment]
Yeah we're unable to update, search for, or install new packages. atom.io/api is down, and Github doesn't seem to have anyone on staff watching it.
